presto-main/src/test/java/io/prestosql/spiller/TestFileSingleStreamSpiller.java

```diff
@@ -70,17 +70,32 @@ public void tearDown()
     public void testSpill()
             throws Exception
     {
-        assertSpill(false);
+        assertSpill(false, false);
     }
 
     @Test
     public void testSpillCompression()
             throws Exception
     {
-        assertSpill(true);
+        assertSpill(true, false);
     }
 
-    private void assertSpill(boolean compression)
+    @Test
+    public void testSpillEncryption()
+            throws Exception
+    {
+        // Both with compression enabled and disabled
```

Comment:
This comment isn't needed, and is a bit misleading since it applies to this test method body and the next

Reply:
Will remove.